cmd/clusterctl/pkg/client/init.go

```diff
@@ -130,6 +95,93 @@ func (c *clusterctlClient) Init(options InitOptions) ([]Components, error) {
 	return aliasComponents, nil
 }
 
+// Init returns the list of images required for init.
```

Comment:
nit: `InitImages`

Reply:
well, seems like we need a follow-up PR :)

Reply:
I added a commit with your other suggestion